docs/release-notes/june-2023.md

```diff
@@ -0,0 +1,233 @@
+This change resolves an issue where dataflow was not correctly accounted for when tainted values flowed through field definitions in class/object definitions. For instance, in Kotlin or Scala, singleton objects are commonly used to encapsulate executable logic, where each field definition behaves like a statement during object initialization. In order to handle this scenario, we have introduced an additional step to analyze a sequence of field definitions as a sequence of statements for taint analysis. This enhancement allows us to accurately track tainted values during object initialization. ([#7742](https://github.com/returntocorp/semgrep/pull/7975)) 
+* Allow any characters in file paths used to create dotted rule IDs. File path characters that aren't allowed in rule IDs are simply removed. For example, a rule whose ID is `my-rule` found in the file `hello/@world/rules.yaml` becomes `hello.world.my-rule`. ([#8057](https://github.com/returntocorp/semgrep/pull/8057))
+* Fixed a typing issue with Go. Previously, the pattern `($VAR : *tau.rho).$F()` wouldn't produce a match in the following:
+  ```
+  func f() {
+    i_1 := &tau.rho{}
+    i_2 := new(tau.rho)
+  
+    i_1.shift() //miss one
+    i_2.left()  //miss two
+  
+    return 101
+  }
+  ```
+but now Semgrep doesn't miss those two findings. ([#8125](https://github.com/returntocorp/semgrep/pull/8125))
+
+## Semgrep Cloud Platform
+
+### Added
+* You can now rename projects (repositories). This feature may be useful when you change the name of your repository in your source code manager. To rename a project from within Semgrep Cloud Platform, do the following:
+    1. Click **[Projects](https://semgrep.dev/orgs/-/projects/)**.
+    2. Click the **<i class="fa-solid fa-gear"></i> gear icon** on the entry of the project you want to rename.
+    3. Click the **<i class="fas fa-ellipsis-h"></i> three-dot icon > Rename project**.
+    4. Enter the name of the new project and click **Rename**.
+    ![Screenshot of repository rename modal](/img/projects-rename-repo.png#md-width)
+* You can now select a default RBAC (role-based access control) role. Refer to [Setting a default role](/semgrep-cloud-platform/user-management/#setting-a-default-role).
+* The Policies page is now the default page used to manage rules. See the [Policies](/semgrep-code/policies) documentation for more information.
+
+
+### Changed
+
+* For organizations with multiple GitHub organizations, you can now add organizations as a newline separated list.
+
+### Removed
+
+* The **Disliked** column has been removed from the Dashboard.
+
+## Semgrep Code
+
+### Added
+
+* Semgrep Pro Engine for Java: Semgrep's taint mode can now relate Java properties and their corresponding getters or setters even when these are autogenerated, so the actual getters or setters are not declared in the sources. 
+* Findings page: Added a badge to identify security-related findings.
```

Comment:
@s-santillan you may know something I don't but I am not aware of this work. Can you attach a screenshot for what this is referring to?

Reply:
Oh! It's this one: 
<img width="604" alt="Snag_1d0a3ef" src="https://github.com/returntocorp/semgrep-docs/assets/20766840/9193cfc8-867d-4965-bea9-e9f57ba1613c">

Already added in the docs. Scroll down a bit: https://semgrep.dev/docs/semgrep-code/findings/#understanding-the-findings-page

Reply:
Well I'll be, look at that!